src/zjb.zig

```diff
@@ -306,6 +313,26 @@ fn validateFromJavascriptArgumentType(comptime T: type) void {
     }
 }
 
+fn validateGlobalType(comptime T: type) void {
+    switch (T) {
+        bool, i32, i64, u32, u64, f32, f64 => {},
+        else => @compileError("unexpected type " ++ @typeName(T) ++ ". Supported types here: bool, i32, i64, u32, u64, f32, f64."),
+    }
+}
+
+fn shortTypeNameGlobal(comptime T: type) []const u8 {
+    return switch (T) {
+        bool => "b",
```

Comment:
Going down to one character and using capitalization to indicate length is excessive.  Just use the zig names: `bool`, `i32`, `i64`, etc.